### PR TITLE
testsys: Update testsys commit to `07b9ae8`

### PR DESCRIPTION
**Pull request by ecpullen**
**Issue number:**

Closes #N/A

**Description of changes:**

This PR updates the checkout of testsys to the later version of testsys.
Updating testsys includes:
* Improved status checking with `testsys status`
* Enable the use of conflicting resources
* Sonobuoy test in China partition

This pr will precede a declarative testing pr that introduces `Test.toml`

**Testing done:**

`cargo make test` and other targets work as expected.

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

**Review by etungsten — APPROVED** [review]

**Review by stmcginnis — APPROVED** [review]

File: tools/testsys/src/aws_resources.rs [hunk 1]
```diff
@@ -493,6 +496,7 @@ impl AwsEcs {
             },
             spec: ResourceSpec {
                 depends_on: None,
+                conflicts_with: None,
```
Comment (bcressey) [inline tools/testsys/src/aws_resources.rs:499]: I'd like for `ResourceSpec` to implement the `Default` trait so we can populate these `None` fields automatically via `..Default::default()`.

File: tools/testsys/Cargo.toml [hunk 2]
```diff
@@ -10,19 +10,19 @@ publish = false
...
+bottlerocket-types = { git = "https://github.com/bottlerocket-os/bottlerocket-test-system", rev = "07b9ae8", version = "0.0.1"}
 bottlerocket-variant = { version = "0.1", path = "../../sources/bottlerocket-variant" }
 clap = { version = "3", features = ["derive", "env"] }
 env_logger = "0.9"
 futures = "0.3.8"
 k8s-openapi = { version = "0.15", features = ["v1_20", "api"], default-features = false }
 log = "0.4"
 maplit = "1.0.2"
-model = { git = "https://github.com/bottlerocket-os/bottlerocket-test-system", rev = "021e8d6", version = "0.1"}
+model = { git = "https://github.com/bottlerocket-os/bottlerocket-test-system", rev = "07b9ae8", version = "0.0.1"}
```
Comment (bcressey) [inline tools/testsys/Cargo.toml:21]: We probably want to rename `model` to something like `bottlerocket-test-model` that doesn't clash with a published crate. `bottlerocket-types` could be `bottlerocket-test-types` also.

File: tools/testsys/Cargo.toml [hunk 3]
```diff
@@ -10,19 +10,19 @@ publish = false
 anyhow = "1.0"
 aws-config = "0.46"
 aws-sdk-ec2 = "0.16"
-bottlerocket-types = { git = "https://github.com/bottlerocket-os/bottlerocket-test-system", rev = "021e8d6", version = "0.1"}
+bottlerocket-types = { git = "https://github.com/bottlerocket-os/bottlerocket-test-system", rev = "07b9ae8", version = "0.0.1"}
```
Comment (arnaldo2792) [inline tools/testsys/Cargo.toml:13]: What needs to be done to track the latest commit/release?

**Review by bcressey — APPROVED** [review]
No blockers, just a couple suggestions for future releases that jumped out at me during review.

**Pull request merged**